### ISSUE TITLE
Slack integration "as_user" fails

### ISSUE THREAD

**Issue by rion18** [DataFire/Integrations#34]
Hi! I'm using @datafire/slack to send messages on behalf of a bot I've built on my own Slack team. 

**What happens**
I'm using the integration's `chat_postMessage` action, with the following payload:

```
 {
    "channel": "@myUser",
    "text": "awesomeText"
  }
```

And it works. However, the mssage is sent through the "Slackbot" user on Slack. I wish to send this message on behalf of the App's bot. When trying this:

```
 {
   "as_user": true,
    "channel": "@myUser",
    "text": "awesomeText"
  }
```

OR

```
 {
   "as_user": false,
    "channel": "@myUser",
    "text": "awesomeText"
  }
```

The request fails with:
```
_http_outgoing.js:642
    throw new TypeError('First argument must be a string or Buffer');
    ^
```
It seems as it's not letting the payload have a boolean value?

**Comment by rion18** [comment]
I've tried with other commands such as `mrkdwn` and I can confirm that FormData with non String values is failing, potentially for every openApi integration?

**Comment by bobby-brennan** [comment]
Thanks for the report! This is now fixed in `2.26.1`
